https://issues.redhat.com/browse/ACM-29169---adding the announcement (#8623)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

[#virt-new-features]
== Virtualization

> - Fine-grained Role-Based Access Control (RBAC) for virtualization is now generally available. Fine-grained (RBAC) provides administrators with the precision to manage permissions across managed clusters with greater security and flexibility.

Reviewer: In Kubernetes doc, "Role-Based Access Control" is lowercase.

```suggestion
- Fine-grained role-based access control (RBAC) for virtualization is now generally available. Fine-grained RBAC provides administrators with the precision to manage permissions across managed clusters with greater security and flexibility.
```

- The {acm-short} fleet management interface now includes specialized roles designed specifically for managing virtual machines across clusters, which extend standard virtualization capabilities, offering specific access levels for fleet-wide operations.

> - With the new `MulticlusterRoleAssignment` custom resource, providing a declarative method for managing Role-Based Access Control (RBAC) across your virtual machines. Administrators can now centralize permission logic on the hub cluster and propagate it to managed clusters.

Reviewer: Let's remove the subordinating clause and lead with the subject. I was little confused with the note starting with, "With the new `MultiClusterRoleAssignment` custom resource, providing a declarative method for managing RBAC across your virtual machines." I figured the period was intended to be a comma. 

```suggestion
With the new `MultiClusterRoleAssignment` custom resource, you can manage role-based access control (RBAC) for virtual machines. As an administer, use the declarative method to centralize permission logic on the hub cluster for propagation with managed clusters.
```

Author: So, I don't like to use "this" -- I changed it here. The reason is that "this" is self-demonstrative, meaning they already know what "this" is. I try to avoid that unless needed. I can make the other changes, but I also wanted "new" in there, so I will commit and edit a bit @dockerymick -- taking out the confusing first line, which was a mistake edit.
